hello-data-portal/hello-data-portal-api/src/main/java/ch/bedag/dap/hellodata/portal/initialize/service/DefaultUserInitializer.java

```diff
@@ -71,27 +71,48 @@ public class DefaultUserInitializer {
     @Transactional(propagation = Propagation.REQUIRES_NEW)
     public boolean initDefaultUsers() {
         boolean defaultUsersInitiated = false;
-        if (defaultAdminProperties.getEmail() == null) {
+        String username = defaultAdminProperties.getUsername();
+        String email = defaultAdminProperties.getEmail();
+        if (email == null) {
             log.warn("No default admin properties set, omitting");
             return defaultUsersInitiated;
         }
+        List<UserRepresentation> allUsersFromKeycloak = getAllUsersFromKeycloak();
+        Optional<UserRepresentation> userByEmail = allUsersFromKeycloak.stream().filter(user -> user.getEmail().equalsIgnoreCase(email)).findFirst();
+        Optional<UserRepresentation> userByUsername = allUsersFromKeycloak.stream().filter(user -> user.getUsername().equalsIgnoreCase(username)).findFirst();
+        boolean theSameUser = userByEmail.isPresent() && userByUsername.isPresent() && userByUsername.get().getId().equalsIgnoreCase(userByEmail.get().getId());
+
+        if (!theSameUser) {
+            log.info("Users fetched from the keycloak:");
+            allUsersFromKeycloak.forEach(
```

Comment:
Maybe only log the userByMail and userByUsername users instead of all keycloak-users

Reply:
sure, done